src/write/line.rs

```diff
@@ -1052,6 +1093,12 @@ mod convert {
                         timestamp: from_file.timestamp(),
                         size: from_file.size(),
                         md5: *from_file.md5(),
+                        source: match from_file.source_attr() {
+                            Some(source) => {
+                                Some(LineString::from(source, dwarf, line_strings, strings)?)
+                            }
+                            None => None,
+                        },
```

Comment:
This also needs to set `program.file_has_source` above where `program.file_has_md5` is set.